generators/exercises/protein_translation.py

```diff
@@ -0,0 +1,19 @@
+FUNC_PROTO = """\
+#include "vendor/unity.h"
+
+#define BUFFER_SIZE 80
+
+extern void proteins(char *buffer, const char *strand);
+"""
+
+def gen_func_body(prop, inp, expected):
+    strand = inp["strand"]
+    if expected.__class__ == dict:
+        expected = ''
+    else:
+        expected = ''.join(map(lambda amino_acid: amino_acid + '\\n', expected))
```

Comment:
I find list comprehensions more readable, but that's certainly a matter of taste:
```suggestion
        expected = ''.join(amino_acid + '\\n' for amino_acid in expected)
```